### PR TITLE
Adds link to VaultPress for backup and scan upsell pages

### PR DESCRIPTION
**Pull request by ChaosExAnima**
#### Changes proposed in this Pull Request

* This modifies the Backup and Scan upsell pages to send users to the VaultPress dashboard.
* Adds tracks to the Backup upsell page.
* Modifies the Scan upsell component to rely on hooks instead of `connect`.

#### Testing instructions

* Create a site with VaultPress.
* Visit the site in Cloud, both the scanner and backup pages.

Backups:
Screen Shot 2020-05-07 at 5 34 28 PM

Scan:
Screen Shot 2020-05-07 at 4 55 08 PM

Fixes 1143508703416848-as-1174802925647065.

### CODE REVIEW

File: client/landing/jetpack-cloud/sections/backups/backup-upsell/index.tsx [hunk 1]
```diff
@@ -13,39 +13,70 @@ import DocumentHead from 'components/data/document-head';
...
-const BackupsUpsellPage: FunctionComponent = () => {
+const BackupsVPActiveBody: FunctionComponent = () => {
 	const translate = useTranslate();
-	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const dispatch = useDispatch();
+	return (
+		<Upsell
+			headerText={ translate( 'Your site has VaultPress' ) }
+			bodyText={ translate(
+				'Your site already is backed up by VaultPress. You can find a link to your VaultPress dashboard below.'
```
Comment (mattgawarecki) [inline client/landing/jetpack-cloud/sections/backups/backup-upsell/index.tsx:31]: This is a small detail, but I think the copy here would read better in English as "Your site **is already** backed up by VaultPress." (Also applies to `scan/upsell.jsx`)

Comment (keoshi) [reply client/landing/jetpack-cloud/sections/backups/backup-upsell/index.tsx:31]: Nice catch!

Comment (ChaosExAnima) [reply client/landing/jetpack-cloud/sections/backups/backup-upsell/index.tsx]: Resolved in 6341e6d.

**Comment by ChaosExAnima** [comment]
@keoshi - That's really peculiar, I definitely see the backups logo. Is anyone else able to repro?

I do see the jump, though. I'm going to test and see what's going on there.

**Comment by rcanepa** [comment]
I just tested this and I saw the logo.

image

**Comment by ChaosExAnima** [comment]
@rcanepa - For Backups, not Scan. :)

**Comment by keoshi** [comment]
I see the logo on Scan, not on Backup. Not sure if that makes a difference.

**Comment by ChaosExAnima** [comment]
Okay, I fixed the loading flash. Turns out the backup state is `uninitialized` but the scan state is `null`. @rcanepa, we really should have some shared types for both state machines!

@keoshi - Only thing that's different is backups uses an `img` tag, and scan directly embeds the SVG. Question: for the regular upsell page on stage, not the VP one, do you see the logo?

**Comment by rcanepa** [comment]
> @rcanepa - For Backups, not Scan. :)

I need more coffee. I retested it for Backup and I saw the logo.
image

@keoshi , can you show us the URL of the broken image?

**Comment by rcanepa** [comment]
> Okay, I fixed the loading flash. Turns out the backup state is uninitialized but the scan state is null. @rcanepa, we really should have some shared types for both state machines!

Definitely!

**Comment by mattgawarecki** [comment]
Late to the party here, but I'm seeing correct logos for both Backup and Scan with no loading flash. 👍

**Pull request merged**